shared/components/NavigationBar/index.scss

```diff
@@ -1,57 +1,150 @@
-.NavigationBar {
-  background: $grey;
-
-  li {
-    list-style-type: none;
+body {
```

Comment:
This is for review purposes as we do not use this component. I will remove this once +1ed.